Microsoft.Kiota.Abstractions.v3.ncrunchsolution

```diff
@@ -0,0 +1,8 @@
+﻿<SolutionConfiguration>
```

Comment:
remove this file

Reply:
I was thinking whether to include it or not, it doesn't hurt and helps with ncrunch while it doesn't interfere where there is no ncrunch.